documentation/docs/configuration/yaml_config.md

```diff
@@ -240,3 +240,17 @@ of an alerting profile from the env var `ALERTING_PROFILE_VALUE`.
 ```
 
 > :warning: Values you pass into a configuration as environment variables must not contain the `=` character.
+
+### Newlines in variables
+
+All line breaks in variables are escaped.
```

Comment:
Do we support all types of line breaks in yaml now (https://yaml-multiline.info/) or a subset? (TIL there's a lot more options to do multi lines..)
If all would be nice to add a few tests with > multilines and quoted and plain multilines. 
If not please describe explicitly what multilines are okay to use :)

Reply:
As long as the yaml parser supports it, we support it. The string we get from yaml is processed. Our code has no extra logic to check how the user defined the text. 
Not sure if it makes sense to include all options yaml gives us. To do this would include at least nine checks.

If we want to do this, we should add a unit-test for the parser itself and verify input & output.